### PR TITLE
add support for soundcloud urls

### PR DESCRIPTION
**Pull request by derUli**

### CODE REVIEW

**Comment by dereuromark** [comment]
Please make sure your IDE doesn't kill any whitespace, as per cs standard.
Let's try to keep the changes minimal to the functional ones, no whitespace noise etc.

**Comment by derUli** [comment]
@dereuromark I just used your .editorconfig in my IDE.
I will fix the CI build.

**Comment by derUli** [comment]
CI build is now green.

**Comment by derUli** [comment]
I look if I can disable the auto formatting on save in this project for future commits

File: src/Data/stubs.php [hunk 1]
```diff
@@ -1515,8 +1526,8 @@
 		'embed-height' => '320',
 		'flashvars' => 'config=http://www.mehr-schbass.de/videos/xml-config.php?id=$2-embed',
 		'id' => '$2',
-		//'name' => '$3',
-	],
+	//'name' => '$3',
+	]
```
Comment (dereuromark) [inline src/Data/stubs.php]: Trailing comma here is not supposed to be removed. Also a few other changes in whitespace look bad.
Rest seems fine.

Comment (derUli) [reply src/Data/stubs.php]: Whats the sense of a comma if this is the last item of the array?

The whitespaces are just auto formatted code as configured in your .editorconfig file.

Comment (dereuromark) [reply src/Data/stubs.php]: SlevomatCodingStandard.Arrays.TrailingArrayComma
For smaller diffs in PRs (the previous line wouldnt show as modified etc).
I am fairly certain a few things went a bit too far. But I guess I can revert those after the fact.

Comment (derUli) [reply src/Data/stubs.php]: Im honestly: This is the first time I saw an array with a trailing comma in PHP code.

I stop using automatic code intendation in your repository.
Would you please create an issue for the oEmbed fallback?

**Comment by dereuromark** [comment]
https://github.com/dereuromark/media-embed/issues/45

**Pull request merged**